third_party/Makefile

```diff
@@ -15,3 +15,9 @@ clean:
 
 distclean:
 	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) distclean $(NL) )
+
+init:
+	@ $(foreach i, $(DIRS), $(MAKE) -C $(i) $(i) $(NL) )
```

Comment:
Which 3rd-party projects depend on this definition?

Reply:
All the projects in line 4: 

```
DIRS = openenclave musl mbedtls libc-test gcompat
```

It clones those 3rd party projects; subsequent calls won't do anything if the projects have already been cloned